react/features/jwt/TokenData.js

```diff
@@ -0,0 +1,68 @@
+/* @flow */
+
+import jwtDecode from 'jwt-decode';
+
+/**
+ * Represent the data parsed from the JWT token
+ */
+export default class TokenData {
+
+    /**
+     * Object representing callee data.
+     */
+    callee: Object;
+
+    /**
+     * Object representing caller data.
+     */
+    caller: Object;
+
+    /**
+     * Contains group. Taken from the token payload.
+     */
+    group: string;
+
+    /**
+     * Indicates whether the user is a guest.
+     */
+    isGuest: boolean;
```

Comment:
This should not be here. It's not part of the token itself, this class represents the token. The user being a guest or not is a decision made outside.